src/Mutation/CommentDelete.php

```diff
@@ -105,8 +105,8 @@ public static function mutate_and_get_payload() {
 			} else {
 				// Get the current user id
 				$current_user_id = absint( get_current_user_id() );
-				// If the current user ID is the same as the comment author's ID, then the
-				// current user is the comment author and can delete the comment
+				// If the current user ID is the same as the commenter's ID, then the
```

Comment:
Cosmetic. I think I was trying to keep consistent language through out the code base.